src/fixtures/test-fixtures.ts

```
@@ -0,0 +1,65 @@
+  },
+
+  // API client fixture
+  apiClient: async ({ request }, use) => {
+    const config = ConfigManager.getInstance();
+    const apiUrl = config.getEnvironment().apiUrl || 'https://jsonplaceholder.typicode.com';
+    const apiClient = new ApiClient(request, apiUrl);
+    await use(apiClient);
+  },
+
+  // Data generator fixture
+  dataGenerator: async ({}, use) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)